price-estimator/src/filter.rs

```diff
@@ -78,15 +78,17 @@ async fn estimate_buy_amount<T>(
 }
 
 async fn get_markets<T>(
-    _token_pair: TokenPair,
+    token_pair: TokenPair,
     _query: QueryParameters,
-    _orderbook: Arc<Orderbook<T>>,
+    orderbook: Arc<Orderbook<T>>,
 ) -> Result<impl warp::Reply, Infallible> {
-    // TODO: implement
-    Ok(warp::reply::json(&MarketsResult {
-        asks: Vec::new(),
-        bids: Vec::new(),
-    }))
+    let transitive_orderbook = orderbook.get_pricegraph().await.transitive_orderbook(
+        token_pair.sell_token_id,
+        token_pair.buy_token_id,
+        None,
```

Comment:
Should we use a spread?

    /// This method optionally accepts a spread that is a decimal fraction that
    /// defines the maximume transitive order price with the equation:
    /// `first_transitive_price + first_transitive_price * spread`. This means
    /// that given a spread of 0.5 (or 50%), and if the cheapest transitive
    /// order has a price of 1.2, then the maximum price will be `1.8`.

Reply:
Since the nodejs price estimator didn't have this I haven't set it here either.

Reply:
Originally, I thought that the runtime for calculating all transitive orders for a token pair would be high, so the spread was supposed to limit this. It turned out not to be the case, so I think we can leave it out.